docs/01-get-started/loading-data-from-unified-graphql-schema.mdx

```diff
@@ -0,0 +1,135 @@
+---
+title: Loading data from unified GraphQL schema
+description:
+  Explore the integration of a unified GraphQL schema in a Front-Commerce route
+  using Remix. This guide delves into the practical aspects of querying and
+  managing data through GraphQL, demonstrating the setup, execution, and
+  optimization of data requests. Understand the synergistic relationship between
+  Front-Commerce and Remix, with a focus on leveraging GraphQL for efficient
+  data handling and page rendering.
+sidebar_position: 4
+---
+
+<p>{frontMatter.description}</p>
+
+## Prerequisites
+
+Before diving into the code, ensure you have the following prerequisites in
+place:
+
+- A Front-Commerce project configured and ready to use
+- Basic knowledge of
+  [Remix routes](https://remix.run/docs/en/main/discussion/routes)
```

Comment:
**nitpick:** I think it's better to link to our own content explaining the previous steps

**suggestion:** replace the link with our how-to guide explaining routes basics (it itself links to further official / advanced resources)

```suggestion
- Basic knowledge of
  [Remix routes](/docs/3.x/get-started/your-first-route)
```